tests/helpers/migrations/20240115223956_add_sample_posts.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
```

Comment:
You accidentally pushed the `migrations` folder. This is just a copy for the tests, I'll add it to the `.gitignore` so your git doesn't show changes. Remove all these migration files from the PR